### PR TITLE
Added kustomize plugins locally info

### PR DESCRIPTION
**Pull request by mvazquezc**

### CODE REVIEW

File: ztp/policygentool/kustomize-plugins-locally.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+This document covers how to run the [PolicyGenTool Generator plugins](https://github.com/openshift-kni/cnf-features-deploy/tree/master/ztp) locally.
+
+1. Get Kustomize and install it, you can get it from the [releases page](https://github.com/kubernetes-sigs/kustomize/releases/).
+2. Extract plugins from the PolicyGenTool image:
+
+    ~~~sh
+    mkdir /tmp/ztp-kustomize/
+    podman cp $(podman create --name policgentool --rm quay.io/redhat_emp1/ztp-site-generator:4.10.0-1):/kustomize /tmp/ztp-kustomize/
+    podman rm -f policgentool
```
Comment (e-minguez) [inline ztp/policygentool/kustomize-plugins-locally.md:11]: What is this for?

File: ztp/policygentool/kustomize-plugins-locally.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# **Run PolicyGenTool Kustomize Plugins Locally**
+
+This document covers how to run the [PolicyGenTool Generator plugins](https://github.com/openshift-kni/cnf-features-deploy/tree/master/ztp) locally.
+
+1. Get Kustomize and install it, you can get it from the [releases page](https://github.com/kubernetes-sigs/kustomize/releases/).
+2. Extract plugins from the PolicyGenTool image:
+
+    ~~~sh
+    mkdir /tmp/ztp-kustomize/
+    podman cp $(podman create --name policgentool --rm quay.io/redhat_emp1/ztp-site-generator:4.10.0-1):/kustomize /tmp/ztp-kustomize/
```
Comment (e-minguez) [inline ztp/policygentool/kustomize-plugins-locally.md]:
```suggestion
mkdir -p ~/.config/kustomize/plugin
podman cp $(podman create --name policgentool --rm quay.io/redhat_emp1/ztp-site-generator:4.10.0-1):/kustomize/plugin/ran.openshift.io ~/.config/kustomize/plugin/
```

Comment (e-minguez) [reply ztp/policygentool/kustomize-plugins-locally.md]: Not sure if I'm missing something but maybe you can skip the /tmp/ stuff and copy the files directly (?)

Comment (mvazquezc) [reply ztp/policygentool/kustomize-plugins-locally.md]: You're right. Committed the suggestion.

File: ztp/policygentool/kustomize-plugins-locally.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+# **Run PolicyGenTool Kustomize Plugins Locally**
+
+This document covers how to run the [PolicyGenTool Generator plugins](https://github.com/openshift-kni/cnf-features-deploy/tree/master/ztp) locally.
+
+1. Get Kustomize and install it, you can get it from the [releases page](https://github.com/kubernetes-sigs/kustomize/releases/).
+2. Extract plugins from the PolicyGenTool image:
+
+    ~~~sh
+    mkdir /tmp/ztp-kustomize/
```
Comment (e-minguez) [inline ztp/policygentool/kustomize-plugins-locally.md]: This is no longer needed then, right?

File: ztp/policygentool/kustomize-plugins-locally.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+mkdir -p ~/.config/kustomize/plugin
+podman cp $(podman create --name policgentool --rm quay.io/redhat_emp1/ztp-site-generator:4.10.0-1):/kustomize/plugin/ran.openshift.io ~/.config/kustomize/plugin/
+    podman rm -f policgentool
+    ~~~
+
+3. Create the plugin directory under your user's home directory:
```
Comment (e-minguez) [inline ztp/policygentool/kustomize-plugins-locally.md]: No longer needed

File: ztp/policygentool/kustomize-plugins-locally.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+    ~~~sh
+    mkdir -p ~/.config/kustomize/plugin
+    ~~~
+
+4. Copy the Kustomize plugins to the plugins directory:
```
Comment (e-minguez) [inline ztp/policygentool/kustomize-plugins-locally.md]: No longer needed

File: ztp/policygentool/kustomize-plugins-locally.md (hunk 4 above)
Comment (mvazquezc) [reply ztp/policygentool/kustomize-plugins-locally.md]: Done.

File: ztp/policygentool/kustomize-plugins-locally.md (hunk 3 above)
Comment (mvazquezc) [reply ztp/policygentool/kustomize-plugins-locally.md]: Removed.

File: ztp/policygentool/kustomize-plugins-locally.md (hunk 5 above)
Comment (mvazquezc) [reply ztp/policygentool/kustomize-plugins-locally.md]: Removed.

**Review by e-minguez — APPROVED** [review]

**Pull request merged**